protocol/contracts/lease/src/contract/mod.rs

```diff
@@ -5,11 +5,13 @@ use sdk::cosmwasm_std::QuerierWrapper;
 
 use crate::lease::{with_lease::WithLease, LeaseDTO};
 
+#[cfg(feature = "contract")]
 pub use self::endpoins::{execute, instantiate, migrate, query, reply, sudo};
 use self::finalize::FinalizerRef;
 
 mod api;
 mod cmd;
+#[cfg(feature = "contract")]
```

Comment:
not sure if we need `contract` feature at the lease at all cause we have `#[cfg(any(feature = "astroport", feature = "osmosis"))]`